kodi_addon_checker/check_dependencies.py

```diff
@@ -42,6 +47,21 @@ def check_addon_dependencies(report: Report, repo_addons: dict, parsed_xml):
                                   % (required_addon, required_version, available_version)))
 
 
+def _get_ignore_list(branch_name):
+
+    if branch_name == "leia":
+        common_ignore_deps.extend(["inputstream.adaptative", "inputstream.rtmp",
+                                   "script.module.cryptodome"])
+        return common_ignore_deps
+
+    elif branch_name == "krypton":
+        common_ignore_deps.extend(["inputstream.adaptative", "inputstream.rtmp"])
```

Comment:
Not a big fan of mutating lists in this way. I would have used `+` I guess. See fore more info: https://stackoverflow.com/a/252711/2859813

Reply:
Just tested it and no it doesn't change with time.
Do you still want me to use the `+`